components/ActiveLink.jsx

```diff
@@ -0,0 +1,14 @@
+import { useRouter } from 'next/router'
+import Link from 'next/link'
+
+export const ActiveLink = ({ text, href }) => {
```

Comment:
The component name is bad because it's a component link agnostic.

`LinkStyled` seems better.